Test/comp/Comprehensions.dfy

```diff
@@ -534,3 +535,65 @@ module NewtypeRangeRegression {
     print k, "\n";
   }
 }
+
+module {:options "/functionSyntax:4"} EnumerationRegression {
+  method Test() {
+    // The Java compiler once converted each candidate Long number to a 32-bit int, which meant
+    // that none of them fell into the range mentioned by the set. Thus, it erroneously computed
+    // the empty set.
```

Comment:
Worth logging these issues as a soundness bug even if it will be immediately fixed here.

Reply:
Done. https://github.com/dafny-lang/dafny/issues/3204